bundles/admin/admin-layereditor/view/AdminLayerForm/VisualizationTabPane/VisualizationTabPane.jsx

```diff
@@ -28,6 +31,9 @@ export const VisualizationTabPane = ({ layer, capabilities, scales, propertyFiel
             { propertyFields.includes(OPACITY) &&
                 <Opacity layer={layer} controller={controller} />
             }
+            { propertyFields.includes(TIMES) &&
```

Comment:
This checks if the functionality _can_ be configured based on the layer type. I think we should also check if the layer we are editing has a timeseries before showing this. However if a layer loses it's timeseries the metadata can't be unlinked in such case.

Check where the timeseries times are stored on the layer. I think they should be on layer.times or layer.capabilities.times. If the layer has a timeseries (==has the times array) OR if the layer.options.timeseries object is present -> show the control. Otherwise it can be left out of the UI.

Reply:
Makes sense. Thanks for detailed explanation.